nix/haskell-pins.nix

```diff
@@ -116,6 +116,20 @@ let
       };
     };
 
+    http-client = {
+      src = fetchgit {
+        url = "https://github.com/wireapp/http-client";
+        rev = "e6beaccdcc8fac892d2437ebbff029fef3551a13";
+        sha256 = "sha256-z47GlT+tHsSlRX4ApSGQIpOpaZiBeqr72/tWuvzw8tc=";
+      };
+      packages = {
+        "http-client" = "http-client";
+        "http-client-tls" = "http-client-tls";
+        "http-client-openssl" = "http-client-openssl";
+        "http-conduit" = "http-conduit";
+      };
+    };
```

Comment:
should I open a PR against our master so this is visible better?

Reply:
Its actually here: https://github.com/wireapp/http-client/pull/5
I am waiting for the CI to be green and us to verify that bots work again. Once that happens we can merge that PR and update the revision here.